### PR TITLE
Exceptions typos/grammar updates

### PR DESCRIPTION
**Pull request by davidchase**
i tried to maintain as much as the original language as possible please let me know your thoughts

### CODE REVIEW

File: docs/exceptions.md [hunk 1]
```diff
@@ -137,83 +137,83 @@ treated formally or informally as **expected failures**.
...
 `throw` for expected failure. But if we do that we'll catch not only errors that represent
 expected failures, but also random errors that represent bugs. This is bad for two reasons:
 
-1. we ruin nice debugging experience (debugger will no longer pause etc);
+1. we ruin nice debugging experience (the debugger will no longer pause etc);
 2. in our code that is supposed to handle expected failures we would need to also
    handle unexpected failures (which is generally imposible as shown in the next section).
 
-If throw is used for expected failures in some API, we should wrap into `try..catch` as little code as
+If a `throw` is used for expected failures in some APIs, we should wrap `try..catch` into as little code as
```
Comment (rpominov) [inline docs/exceptions.md]: Should't "into" be before `try..catch` ? I was trying to say than we wrap should wrap "as little code as possible" inside `try..catch`

File: docs/exceptions.md [hunk 2]
```diff
@@ -224,31 +224,38 @@ Conclusion: we might want to catch unexpected errors in Node, but there are plen
 
 ## Unexpected failures in browser
 
-In case of a browser restarting (reloading the page) usually considered as an awful behavior from
-the UX point of view, so it might be not an option. We may choose not to restart in a hope of
-providing a better UX at a risk of leaking inconsistent state to the database etc. Some bugs are
+Just as in Node we could "restart" the browser or reload the page. However, that option is usually considered as an awful behavior from
```
Comment (rpominov) [inline docs/exceptions.md]: I don't wan't to propose restarting the browser 😄  
I meat that in browser `restarting == reloading the page`

**Comment by rpominov** [comment]
Thank you so much! I didn't realize it was so bad 😄

File: docs/exceptions.md (hunk 2 above)
Comment (davidchase) [inline docs/exceptions.md]: Ya sorry I know what you meant, tried to write a bit differently maybe like this:
```
... Node we could "restart" (reload the page) the browser ...
```

File: docs/exceptions.md (hunk 1 above)
Comment (davidchase) [inline docs/exceptions.md]: Ah yes ok will update that

File: docs/exceptions.md (hunk 2 above)
Comment (rpominov) [inline docs/exceptions.md]: Still sounds like we restarting the browser itself not program that runs inside it (webpage). Maybe:

```
... Node we could "restart" (reload) webpage.
```
?

Comment (rpominov) [inline docs/exceptions.md]: Or how does "... we could restart the program (reload the webpage)" sound?

Comment (davidchase) [inline docs/exceptions.md]: how about this: "Just as in Node we could restart the program by reloading the webpage" tells the reader the restart of the program is done when the webpage reloads?

Comment (rpominov) [inline docs/exceptions.md]: Yea. Sounds good.

**Comment by rpominov** [comment]
I've merged it and fixed remaining stuff in https://github.com/rpominov/fun-task/commit/2fa80d8ee2b2321a0b5a50527cf9995ec81556f4

Thanks again. I very appreciate your effort!

**Comment by davidchase** [comment]
ah ok was just about to update the PR sorry for the delay

**Comment by rpominov** [comment]
No problem 😄

**Pull request merged**